Registration page - updated css classes

hallsi90 opened the pull request:
Installed tailwind v4
Changed the tailwind classes to customized styling classes.

### What does this PR do?

### Description of Task to be completed?

### How should this be manually tested?

### Any background context you want to provide?

hallsi90 committed:
Updated registration page

Installed tailwind v4
Changed the tailwind classes to customized styling classes.

hallsi90 committed:
Merge branch 'develop' into registration-page-ingelinn

Tomatsuppe11211 reviewed (changes requested):
Small fixes:

The button should contain bold font (line 107) according to Figma design.

The phone version is not supposed to have border aroudn the forms. 



Reccomended:

If you like to edit input fields to look more like they do in the design. you can check this link         https://tailwindcss.com/docs/box-shadow

hallsi90 committed:
Update registration-page.html

Updated form colors and shadow.
Removed form border on small screen sizes.

hallsi90 committed:
Merge branch 'develop' into registration-page-ingelinn

hallsi90 commented:
- I've left the button with the classes .primary-button and .primary-button:hover since we agreed on that at our team meeting
- I have removed the form border on smaller screen sizes
- I have updated the border shadows
- I have updated the border and shadow color to #cccccc (like it is in the figma design)

Tomatsuppe11211 reviewed (approved):
Looking good! Nice work!

myrset reviewed (approved):
Page looks nice! This calls for an approval
